packages/map-template/src/components/MapTemplate/MapTemplate.jsx

```diff
@@ -544,6 +547,15 @@ function MapTemplate({ apiKey, gmApiKey, mapboxAccessToken, venue, locationId, p
         setShowRoadNames(showRoadNames);
     }, [showRoadNames])
 
+    /*
+     * React on changes to the searchExternalResults prop.
+     */
+    useEffect(() => {
+        if (mapsindoorsSDKAvailable && searchExternalResults) {
```

Comment:
Is the check for SDK availability needed?

Reply:
Actually not, removed it ✅